packages/instrument-utils/src/storedInstrument/AccountInstrumentSelect/AccountInstrumentSelect.tsx

```diff
@@ -92,7 +136,7 @@ const BankInstrumentMenuItem: FunctionComponent<BankInstrumentMenuItemProps> = (
 };
 
 interface AccountInstrumentOptionProps {
-    instrument: AccountInstrument;
+    instrument: BraintreeAchInstrument | AccountInstrument;
```

Comment:
Thought we would keep adding things here in the future if we allow `BraintreeAchInstrument | AccountInstrument` today. Is there a way to avoid this?

Reply:
@bc-peng i have updated this point. you can take a look